randbp/string.go

```diff
@@ -11,12 +11,42 @@ import (
 // GenerateRandomString.
 const Base64Runes = `ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_+/=`
 
-// GenerateRandomString generates a random string with length [0, maxLength),
-// and all characters limited to runes.
+// RandomStringArgs defines the args used by GenerateRandomString.
+type RandomStringArgs struct {
+	// Required. If MaxLength <= MinLength it will cause panic.
```

Comment:
Shouldn't `MaxLen == MinLen` be fine? I feel like it should only be a problem if `min > max`.

Reply:
The length will be in range of `[MinLength, MaxLength)`, the max end is non-inclusive.